src/core/manager.c

```diff
@@ -2296,17 +2321,18 @@ static int manager_dispatch_signal_fd(sd_event_source *source, int fd, uint32_t
                 break;
 
         case SIGUSR1: {
-                Unit *u;
+                bool b;
 
-                u = manager_get_unit(m, SPECIAL_DBUS_SERVICE);
-
-                if (!u || UNIT_IS_ACTIVE_OR_RELOADING(unit_active_state(u))) {
+                b = manager_dbus_is_running(m, false);
```

Comment:
Would be nicer to get rid of `b` and put the condition directly in the conditional.